README.md

```diff
@@ -283,6 +283,23 @@ output:
 
 ![Example](docs/images/example_data.gif)
 
+### Force Log
+
+Sometimes, you need to log even if level is not enable.
```

Comment:
Yohan say for the doc in open-sources, we always keep simple possible maximum. Here, I think you can juste write something: `we can force to write the log even the logLevel isn't enabled. exemple....`

Reply:
Yes i changed the sentence for a more simple